store/label.go

```diff
@@ -62,14 +65,46 @@ func (s *LabelService) FindLabelKeyList(ctx context.Context, find *api.LabelKeyF
 		}
 
 		ret = append(ret, &labelKey)
+		keymap[labelKey.Key] = &labelKey
 	}
 	if err := rows.Err(); err != nil {
 		return nil, FormatError(err)
 	}
 
+	// Find key values.
+	valueRows, err := tx.QueryContext(ctx, `
+		SELECT
+		    key,
+			value
+		FROM label_value`,
+	)
+	if err != nil {
+		return nil, FormatError(err)
+	}
+	defer valueRows.Close()
+
+	for valueRows.Next() {
+		var key, value string
+		if err := valueRows.Scan(
+			&key,
+			&value,
+		); err != nil {
+			return nil, FormatError(err)
+		}
+		labelKey, ok := keymap[key]
+		if !ok {
+			return nil, common.Errorf(common.Internal, fmt.Errorf("Label value doesn't have a label key, key %q, value %q", key, value))
```

Comment:
Should not capitalize (label instead of Label)

(I thought golint should complain if using VSCode)